geoapi/services/projects.py

```diff
@@ -352,19 +320,17 @@ def removeUserFromProject(database_session, projectId: int, username: str) -> No
         """
         proj = database_session.query(Project).get(projectId)
```

Comment:
Pedantic comment 12: For consistency, wouldn't it be better to use `project` instead of `proj`?